inventory/group_vars/br.yml

```diff
@@ -0,0 +1,20 @@
+---
+
+# See https://github.com/openfoodfoundation/ofn-install/wiki/Setup for more info
+checkout_zone: Brazil
+country_code: BR
+currency: BRL
+locale: pt-BR                  # Default locale for OFN instance
+available_locales: br, pt    # Available locales for OFN instance
+language: pt_BR.UTF-8       # Default language for the server
+language_packages:          # Language packages for the server
+  - language-pack-pt-base
+l10n_repo: https://github.com/rafaelbbraz/l10n_br.git
+timezone: Brasilia
+
+# Used for the LetsEncrypt email setting
+developer_email: frattezi@tremtec.com
+
+# Users listed here will have their key from /files/keys added to the unicorn_user
+users_sysadmin:
+  - frattezi
```

Comment:
I'd recommend having more than one user with super admins access

Reply:
You can use this if you want OFN sysadmins to have access to the server (optional):

```
users_sysadmin:
  - frattezi
  - "{{ core_devs }}"
```